code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -158,8 +158,14 @@
 	name = "guise of Istasha"
 	desc = "An odd amulet formed out of multiple floating parts, strung togethere by forces from another world."
 	icon_state = "eye_medalion"
-	trait = TRAIT_DIGINVIS
 
+/obj/item/clothing/neck/eldritch_amulet/guise/equipped(mob/user, slot)
+	. = ..()
+	user.AddElement(/datum/element/digitalcamo)
```

Comment:
This will trigger when equipped to hand slots

Reply:
Whats the proc for equipped to slot?

Reply:
You might just have to check the `slot` argument, not sure.

Reply:
After looking into it, realised that... there's more issues. Look into the proc you're overriding - the parent type already does this correctly... and this would have the amulet apply both the effects of this and the parent types.

Reply:
Check the slot argument